platform/ios/src/MGLAPIClient.m

```diff
@@ -87,7 +87,7 @@ - (NSURLRequest *)requestForEvents:(NS_ARRAY_OF(MGLMapboxEventAttributes *) *)ev
 }
 
 - (void)setupBaseURL {
-    NSString *testServerURL = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"MGLMetricsTestServerURL"];
+    NSString *testServerURL = [[NSUserDefaults standardUserDefaults] stringForKey:@"MGLMapboxMetricsTestServerURL"];
```

Comment:
Yeah, I'm thinking we should register a default for this default wherever we're currently calling `-[NSUserDefaults registerDefaults:]`.